src/scss/skin-modern/components/_controlbar.scss

```diff
@@ -8,38 +8,46 @@
   @include layout-align-bottom;
 
   background: linear-gradient(to bottom, $color-transparent, $color-background-bars);
-  line-height: 1em;
-  padding: 1em 1em .5em;
+  vertical-align: middle;
 
   .#{$prefix}-controlbar-top,
   .#{$prefix}-controlbar-bottom {
     > .#{$prefix}-container-wrapper {
       display: flex;
-      margin: .5em;
     }
   }
 
-  .#{$prefix}-controlbar-top {
-    .#{$prefix}-ui-label {
-      font-size: .9em;
-    }
+  .#{$prefix}-ui-container > {
+    .#{$prefix}-container-wrapper > * {
+      &:first-child {
+        padding-left: 2rem;
```

Comment:
I just found out that this is an issue if the first issue is `hidden`. (Happens in livestreams without `timeShift`).

A solution would be something like this:
```scss
&:not(.#{$prefix}-hidden) {
  padding-left: 2rem;
}
&:not(.#{$prefix}-hidden) ~ :not(.#{$prefix}-hidden) {
  padding-left: $default-button-badding;
}
```

But i feels so hacky that i don't want this. Again do you have any other idea @protyposis. I was thinking about detecting the first visible item in container JS and add another class but this also feels not perfect.